pages/_middleware.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { NextRequest, NextResponse } from "next/server";
+
+// FIXME - this would be nicer if it was dynamic using `fs.readDirSync('.')
+const SKIP_PATHS = [
+  ".next",
+  "404",
+  "api",
+  "auth",
+  "fonts",
+  "availability",
+  "bookings",
+  "call",
+  "cancel",
+  "event-types",
+  "getting-started",
+  "index",
+  "integrations",
+  "payment",
+  "reschedule",
+  "sandbox",
+  "settings",
+  "success",
+  "team",
+];
+
+const FIXME_LOCALES = ["en", "fr", "it", "ru", "es", "de", "pt", "ro", "nl", "pt-BR", "es-419", "ko", "ja"];
```

Comment:
Wouldn't this work? @KATT 

```suggestion

const { i18n } = require("../next-i18next.config");

// FIXME - this would be nicer if it was dynamic using `fs.readDirSync('.')
const SKIP_PATHS = [
  ".next",
  "404",
  "api",
  "auth",
  "fonts",
  "availability",
  "bookings",
  "call",
  "cancel",
  "event-types",
  "getting-started",
  "index",
  "integrations",
  "payment",
  "reschedule",
  "sandbox",
  "settings",
  "success",
  "team",
];

const FIXME_LOCALES = i18n.locales;
```